PuppeteerSharp.Dom/PuppeteerSharp.Dom.csproj

```diff
@@ -23,7 +23,7 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageId>PuppeteerSharp.Dom</PackageId>
     <PackageReleaseNotes>Upgrade to PuppeteerSharp 19.0</PackageReleaseNotes>
-    <VersionPrefix>5.0.0</VersionPrefix>
+    <VersionPrefix>5.0.40</VersionPrefix>
```

Comment:
Not planning on keeping this updated manually, so keeping this at `5.0.0` is my preference (or whatever the major version number is).

Happy to add a comment in the code to make this clear that you should set the `Version` when building. For simplicity the build run number is used currently when built via `GitHub Actions`.

https://github.com/ChromiumDotNet/PuppeteerSharp.Dom/blob/main/.github/workflows/dotnet.yml#L43

If you are going to build your own local version then you should pass the `Version` to `dotnet build`